keybase/platform_windows.go

```diff
@@ -125,6 +144,41 @@ func CheckRegistryUninstallCode(productID string, log Log) bool {
 	return false
 }
 
+// Read all the runonce subkeys and find the one with "Keybase" in the name.
+func checkRebootPending(wow64 bool, log Log) (bool, error) {
+	var access uint32 = registry.ENUMERATE_SUB_KEYS | registry.QUERY_VALUE
+	if wow64 {
+		access = access | registry.WOW64_32KEY
+	}
+
+	k, err := registry.OpenKey(registry.CURRENT_USER, "SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\RunOnce", access)
+	if err != nil {
+		log.Errorf("Error %s opening RunOnce subkeys", err.Error())
+		return false, err
```

Comment:
I guess key not existing isn't really an error but I guess it's ok since we can't distinguish between that and real error